docs/conf.py

```diff
@@ -34,29 +34,28 @@
     "aliases": {
         "title": "Aliases",
         "description": (
-            "PlasmaPy provides short-named (alias) versions of the most "
-            "common plasma functionality.  These aliases are only given to "
-            "functionality where there is a common lexicon in the community, "
-            "for example `~plasmapy.formulary.frequencies.plasma_frequency` "
-            " has the alias `~plasmapy.formulary.frequencies.wp_`.  All aliases "
-            "in PlasmaPy are denoted with a trailing underscore ``_``."
+            """
+            PlasmaPy provides :term:`aliases` of the most common plasma
+            functionality for user convenience. Aliases in PlasmaPy are
+            denoted with a trailing underscore.
```

Comment:
```suggestion
            denoted with a trailing underscore, e.g. ``alias_``.
```

Also needs a link to the "Aliases" section in the doc guide.